src/app-components/Table/Table.tsx

```diff
@@ -1,40 +1,47 @@
 import React from 'react';
+import type { ReactElement } from 'react';
 
-import { Button, Table } from '@digdir/designsystemet-react';
+import { Button, Spinner, Table } from '@digdir/designsystemet-react';
 import cn from 'classnames';
 import { format, isValid, parseISO } from 'date-fns';
 import { pick } from 'dot-object';
 import type { JSONSchema7 } from 'json-schema';
 
 import classes from 'src/app-components/Table/Table.module.css';
-import type { FormDataObject, FormDataValue } from 'src/app-components/DynamicForm/DynamicForm';
+import { Lang } from 'src/features/language/Lang';
```

Comment:
I thought `app-components/` was supposed to be "separate" from our app-framework? `Lang` uses many hooks that relies on our app-framework 🤔

Reply:
Ai, you're completely right. This was a slip-up 🙈